workflow-dev/tpl/deis-logger-deployment.yaml

```diff
@@ -0,0 +1,68 @@
+#helm:generate helm tpl -d $HELM_GENERATE_DIR/tpl/generate_params.toml -o $HELM_GENERATE_DIR/manifests/deis-logger-deployment.yaml $HELM_GENERATE_DIR/tpl/deis-logger-deployment.yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: deis-logger
+  namespace: deis
+  labels:
+    heritage: deis
+  annotations:
+    helm-keep: "true"
+    component.deis.io/version: {{env "LOGGER_GIT_TAG" | default .logger.dockerTag}}
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
```

Comment:
i think if there will be an issue if two logger instances write to the same redis instance simultaneously.
cc @krancour